Evaluation/Input/Content/Concentration_time_profiles.md

```diff
@@ -1,2 +1,2 @@
-Simulated versus observed concentration-time profiles of all data listed in [Section 2.2.2](#222-clinical-data) are presented below.
+Simulated versus observed concentration-time profiles of all data listed in [Section 2.2.2](#clinical-data) are presented below.
```

Comment:
I extended all sections (also in the static content) with an anchor (s. below) and now referencing the **anchor** instead of the chapter title - which makes it invariant both in case of section number change or section title change.

I wonder if we should replace the section number in TEXT of the reference with the section title
(e.g. `[Section "Clinical Data"]` instead of `[Section 2.2.2]` here)